activiti-cloud-query-service/activiti-cloud-starter-query/src/test/java/org/activiti/cloud/starter/tests/QueryAdminProcessServiceTasksIT.java

```diff
@@ -531,136 +526,226 @@ public void shouldGetServiceTaskIntegrationContextResultById() throws Interrupte
 
     @Test
     public void should_supportLoopInvolvingServiceTasks() {
-        //given
+        //given - first iteration, service task started and completed
         ProcessInstanceImpl process = sendEventsForStartSimpleProcessInstance();
-        String integrationContextId1 = UUID.randomUUID().toString();
-        IntegrationContext integrationContext1 = createIntegrationContext(process, integrationContextId1);
-        sendIntegrationRequestedEvent(integrationContext1);
+        String id_iteration1 = UUID.randomUUID().toString();
+        String id_iteration2 = UUID.randomUUID().toString();
```

Comment:
```suggestion
        String idIteration1 = UUID.randomUUID().toString();
        String idIteration2 = UUID.randomUUID().toString();
```

Camel case instead of snake case.